moped-editor/src/views/staff/StaffForm.js

```diff
@@ -301,8 +293,15 @@ const StaffForm = ({
           <FormControl variant="standard" component="fieldset">
             <FormLabel id="roles-label">Role</FormLabel>
             <Controller
-              as={
```

Comment:
`Controller`'s `as` prop is removed in v7 so we have to refactor with the `render` prop like the other instances of `Controller` in this codebase use.